### PR TITLE
OBPIH-6019 Header with action buttons for Product Sources create/edit form

### PR DESCRIPTION
**Pull request by alannadolny**

### CODE REVIEW

File: src/js/consts/applicationUrls.js [hunk 1]
```diff
@@ -129,8 +129,9 @@ const CATEGORY_URL = {
 
 const PRODUCT_SUPPLIER_URL = {
   base: `${CONTEXT_PATH}/productSupplier`,
+  list: () => `${PRODUCT_SUPPLIER_URL.base}/list`,
   create: () => `${PRODUCT_SUPPLIER_URL.base}/create`,
-  edit: (id) => `${PRODUCT_SUPPLIER_URL.base}/edit/${id}`,
+  edit: (id) => `${PRODUCT_SUPPLIER_URL.base}/create/${id}`,
```
Comment (kchelstowski) [inline src/js/consts/applicationUrls.js:134]: maybe let's do it later (at the end), so we can still edit product sources if we wanted to test something (assuming we click the Edit from the list page)

File: src/js/components/productSupplier/create/CreateProductSupplierTitle.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        {productSupplier
+          ? productSupplier?.name
+          : translate('react.productSupplier.createProductSource.label', 'Create Product Source')}
+      </span>
+      {' '}
+      {productSupplier && (
+        <span className="create-page-title-source-code">
+          (
+          {productSupplier?.code}
+          )
```
[the remark this answers is not in the document]
Comment (alannadolny) [reply src/js/components/productSupplier/create/CreateProductSupplierTitle.jsx:29]: Yeah, that is the reason why it is split into new lines.

File: src/js/consts/applicationUrls.js (hunk 1 above)
Comment (alannadolny) [reply src/js/consts/applicationUrls.js:134]: you can still access the old page manually

Comment (kchelstowski) [reply src/js/consts/applicationUrls.js:134]: I know we can, this is why I wrote this: "(assuming we click the Edit from the list page)".
But let's leave it, as accessing edit through show page should not be "frustrating"

File: src/js/components/productSupplier/create/ProductSupplierForm.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import ProductSupplierFormHeader
+  from 'components/productSupplier/create/ProductSupplierFormHeader';
+import PageWrapper from 'wrappers/PageWrapper';
+
+const ProductSupplierForm = () => (
```
Comment (kchelstowski) [inline src/js/components/productSupplier/create/ProductSupplierForm.jsx:7]: you've forgotten to inject the translations on that page.
```js
useTranslation('productSupplier');
```

**Review by kchelstowski — APPROVED** [review]

**Review by awalkowiak — APPROVED** [review]

**Pull request merged**